### DISCUSSION TITLE
ARM64 Image Version

### DISCUSSION THREAD

**Discussion by MarcelloPercoco** [mat/besticon#106] [Ideas]
Hi.

It's possible to have even the arm64 docker image?

by now i'm building it by myself, and is not a great deal.

THis is what i do in my GitHu action.

`      - name: modify Dockerfile
        run: |
          sed -i 's/amd64/arm64/' ./besticon/Dockerfile
      - name: take besticon version
        working-directory: ./besticon
        run: |
          echo "BESTICON_VERSION=$(cat VERSION)" >> $GITHUB_ENV
      - name: Set up QEMU
        uses: docker/setup-qemu-action@v3
        with:
          platforms: 'arm64'
      - name: Set up Docker Buildx
        uses: docker/setup-buildx-action@v3
        with:
          platforms: 'arm64'`

It's just an example, i'm sure you dosen't need that i tell you what to do.

**Comment by mat** [comment]
https://hub.docker.com/layers/matthiasluedtke/iconserver/latest/images/sha256-8707df0c9bcd6b451d012e4357ca2fc0e379e1ce1de1c2cfcdad416b713333a5?context=repo

![image](https://github.com/user-attachments/assets/d85823de-572b-4420-b50e-55a2d9dd29e6)

**Comment by MarcelloPercoco** [comment]
Wonderful!

Thanks!
